test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationDetailsTypeControllerSpec.scala

```diff
@@ -106,7 +118,7 @@ class OrganisationDetailsTypeControllerSpec extends ControllerSpec {
           )
         )
         mockRegistrationUpdate()
-        val result = controller.displayPage()(getRequest())
+        val result = controller.displayPageNewMember()(getRequest())
 
         status(result) mustBe OK
       }
```

Comment:
We don't check whether the supplied group member org type is valid (since the UI restricts this appropriately). Is this something which we believe we need to do? If so, I can add some protection and appropriate tests to prove that it is working as expected.

Reply:
We’ve had a dev discussion about this and we believe that testing for input which is not permitted by the UI is probably overkill. It would be difficult to know where to draw the line with this kind of testing.

Reply:
As per Slack discussion I'm happy with the UI restricting this as this is the approach we've taken elsewhere :+1: